docs/proposals/gateway-settings.md

```diff
@@ -0,0 +1,183 @@
+# Enhancement Proposal-1775: Gateway Settings
```

Comment:
Does it matter if we still call it Gateway Settings?
Note: Gateway Settings used a lot if the text

Reply:
I decided to use that as a more general term. Those are ultimately what we are configuring, `NginxProxy` is just the name of the resource that contains those settings.

Reply:
lgtm!